lte/gateway/c/core/oai/tasks/amf/nas_proc.cpp

```diff
@@ -469,6 +473,115 @@ int amf_nas_proc_authentication_info_answer(
   OAILOG_FUNC_RETURN(LOG_NAS_AMF, rc);
 }
 
+int amf_decrypt_imsi_info_answer(itti_amf_decrypted_imsi_info_ans_t* aia) {
```

Comment:
Negative result also will be added.

Reply:
added the case of RR with cause# 3

Reply:
is this function covered by any unit tests?